src/sysfiles.c

```diff
@@ -1651,6 +1654,16 @@ Obj OnCharReadHookOutFuncs = 0;/* a list of GAP functions with 0 args */
 Obj OnCharReadHookExcFds = 0;  /* a list of UNIX file descriptors */
 Obj OnCharReadHookExcFuncs = 0;/* a list of GAP functions with 0 args */
 
+// In HPC-GAP this is always active so we can use FreezeStdin
```

Comment:
What is "this" the sentence refers to?

Reply:
If I just looked at this code diff, I think I would have no clue what `FreezeStdin` is doing and why. Perhaps a slightly longer comment could b added somewhere to explain that? Perhaps based on the PR description.